BUILD

```diff
@@ -77,17 +80,7 @@ compilation_database(
     # Unfortunately, compilation_database does not support filesets yet,
     # so expand them here manually.
     # https://github.com/grailbio/bazel-compilation-database/issues/84
-    targets = [
-        "//common/tools:verible-patch-tool",
-        "//verilog/tools/diff:verible-verilog-diff",
-        "//verilog/tools/formatter:verible-verilog-format",
-        "//verilog/tools/kythe:verible-verilog-kythe-extractor",
-        "//verilog/tools/lint:verible-verilog-lint",
-        "//verilog/tools/obfuscator:verible-verilog-obfuscate",
-        "//verilog/tools/preprocessor:verible-verilog-preprocessor",
-        "//verilog/tools/project:verible-verilog-project",
-        "//verilog/tools/syntax:verible-verilog-syntax",
-        "//verilog/tools/ls:verible-verilog-ls",
+    targets = BINARIES + [
```

Comment:
Thanks!
I think I'll try to retry the new compilation db thing later today that should work with filegroups.
I had to roll back my earlier attempt to this last year because of some centos incompatibility, but since we changed bazel versions, maybe this is not an issue anymore.